README.md

```diff
@@ -111,40 +112,31 @@ for i in range(1000000):
     gui.show()
 ```
 
-If Taichi is properly installed, you should get the animation below 🎉:
-
-<a href="https://github.com/taichi-dev/taichi/blob/master/python/taichi/examples/simulation/fractal.py#L1-L31"> </a><img src="https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/fractal_small.gif" height="270px">
```

Comment:
Shall we keep the animation here?  `</a><img src="https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/fractal_small.gif" height="270px">` I think it's fine to duplicate.